app/views.py

```diff
@@ -30,14 +30,14 @@ def isLoggedIn():
     return True
 
 
-def getUser():
+def getUser(username):
     if not isLoggedIn():
         return {'error': "Not logged in."}
 
     ts = pymesync.TimeSync(baseurl=app.config['TIMESYNC_URL'],
                            test=app.config['TESTING'],
                            token=session['token'])
-    user = ts.get_users(username=session['username'])
+    user = ts.get_users(username=username)
 
     return user
```

Comment:
@subnomo We don't need to flash a detailed, technical message to the user. We can just flash a general error message to the user and then print the error to the console or a logfile inside get_user() before the empty dict is returned. That's what you're doing already at line 99.